backends/bevy_picking_raycast/Cargo.toml

```diff
@@ -13,7 +13,7 @@ resolver = "2"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-bevy = { version = "0.10", default-features = false, features = [
+bevy = { version = "0.11", default-features = false, features = [
     "bevy_render",
 ] }
 bevy_mod_raycast = "0.8"
```

Comment:
This needs to be updated to 0.9 for Bevy 0.11 support

Reply:
You're correct ; for precision, bevy_mod_raycast is currently patched to my own branch so it should be working right now (but not in its optimal state indeed)

Reply:
ah yes,
```toml
bevy_mod_picking = { version = "0.14.0", default-features = false, features = ["backend_raycast"], git = "https://github.com/Vrixyz/bevy_mod_picking.git", branch = "bevy-0.11-compatibility" }

[patch.crates-io]
bevy_mod_raycast = { git = "https://github.com/Vrixyz/bevy_mod_raycast.git", branch = "bevy-0.11-compatibility" }
```
appears work thanks

Reply:
I had to also add `bevy_eventlistener = { git = "https://github.com/aevyrie/bevy_eventlistener.git", branch = "main" }` but my app's functionality related to this crate is now working perfect in Bevy 0.11